sdl_android/src/main/java/com/smartdevicelink/api/SdlManager.java

```diff
@@ -141,28 +141,29 @@ protected void initialize(){
 		this.permissionManager = new PermissionManager(_internalInterface);
 		this.permissionManager.start(subManagerListener);
 
+		this.videoStreamingManager = new VideoStreamingManager(_internalInterface);
+		this.videoStreamingManager.start(subManagerListener);
```

Comment:
I'm wondering if it makes sense to create the video streaming manager for every app. It seems as if the most responsible way to handle this would be to only create and start the manager when requested by the developer.

Reply:
Or we could do it based on AppType. If its nav or proj, then we could start it. same w/ the audiostreammanager

Reply:
Yea that's a good idea. Though I'm not sure if audio streaming is limited to those two app types, something need to check into